src/components/alarm/AlarmController.cpp

```diff
@@ -102,14 +98,13 @@ void AlarmController::SetOffAlarmNow() {
 }
 
 void AlarmController::StopAlerting() {
-  systemTask->PushMessage(System::Messages::StopRinging);
-
   // Alarm state is off unless this is a recurring alarm
   if (recurrence == RecurType::None) {
     state = AlarmState::Not_Set;
   } else {
-    state = AlarmState::Set;
```

Comment:
I'm unclear why is this state setting removed? Doesn't it do anything?

Reply:
The state is also set in `ScheduleAlarm()`